src/openfermion/transforms/__init__.py

```diff
@@ -43,3 +43,6 @@
                              weyl_polynomial_quantization)
 from ._remove_symmetry_qubits import (symmetry_conserving_bravyi_kitaev,
                                       edit_hamiltonian_for_spin)
+
+from ._qubit_tapering_from_stabilizer import (reduce_number_of_terms,
```

Comment:
move import into alphabetical order.

Reply:
Hi Nicholas,

Thanks for the review, I'm happy to see that most of the things are in agreement with the code standards.

The __init__ is in fact wrong. We originally put these code in transformations, but at the end decided to move it to utils, hence the transforms init is wrong, and the one in utils must be updated.
This will be done for the merge.

Best,
Xavi